transformers/events/new_cdp/transformer_test.go

```diff
@@ -26,13 +28,25 @@ import (
 )
 
 var _ = Describe("NewCdp Transformer", func() {
-	var transformer = new_cdp.Transformer{}
+	var (
+		transformer = new_cdp.Transformer{}
+		db          = test_config.NewTestDB(test_config.NewTestNode())
```

Comment:
I think we probably want to clean the test db in a `BeforeEach` for these tests where we're now initializing a `db`